geopandas/geodataframe.py

```diff
@@ -5,6 +5,7 @@
 import pandas as pd
 from pandas import DataFrame, Series
 from pandas.core.accessor import CachedAccessor
+from pandas.core.indexing import _iLocIndexer
```

Comment:
I am not sure this is a wise move. I know that we occasionally depend on pandas private functions but I'd like to avoid that if there's another possible way (not that I have one to offer now).